src/pages/LendPage/LendDetailPage.tsx

```diff
@@ -0,0 +1,1314 @@
+                      Oracle:
+                    </Box>
+                    <Box fontSize={'14px'} fontWeight={'500'} color={'#ebecf2'}>
+                      Chainlink
+                    </Box>
+                  </Box>
+                </Box>
+                <Box flex={1}>
+                  <Box
+                    padding={'20px'}
+                    display={'flex'}
+                    alignItems={'center'}
+                    gridGap={'4px'}
+                    borderBottom={'1px solid #32394d'}
+                    sx={{
+                      justifyContent: {
+                        xs: 'space-between',
+                        sm: 'space-between',
+                        md: 'flex-start',
+                      },
+                    }}
+                  >
+                    <Box fontSize={'14px'} fontWeight={'500'} color={'#c7cad9'}>
+                      Whitelist:
+                    </Box>
+                    <Box fontSize={'14px'} fontWeight={'500'} color={'#ebecf2'}>
+                      Yes
+                    </Box>
+                  </Box>
+                </Box>
+              </Box>
+            </Box>
+          </Box>
+          <Box
+            flex={'1'}
+            borderRadius={'20px'}
+            bgcolor={'#232734'}
+            display={'flex'}
+            flexDirection={'column'}
+            sx={{ minWidth: { xs: '55%', sm: '35%' } }}
+          >
+            <Box
+              height={'70px'}
+              borderBottom={'solid 1px #32394d'}
+              display={'flex'}
+              justifyContent={'space-between'}
+              alignItems={'center'}
+            >
+              <Box
+                px={'30px'}
+                fontSize={'18px'}
+                fontWeight={'500'}
+                lineHeight={'1'}
+              >
+                QUICK Statistics
+              </Box>
+              <Box mr={'30px'} display={'flex'} fontSize={'14px'}>
+                <CustomSelect arrowcolor={'#428afa'}>
+                  <SmOption value={'QUICK'}>QUICK</SmOption>
+                  <SmOption value={'123123'}>123123</SmOption>
+                  <SmOption value={'12313212'}>12313212</SmOption>
+                </CustomSelect>
+              </Box>
+            </Box>
+            <Box flex={1} pb={'16px'} display={'flex'} flexDirection={'column'}>
+              <Box flex={1} position={'relative'} pt={'10px'} paddingX={'30px'}>
+                <Box
+                  ml={'-5px'}
+                  mr={'auto'}
+                  padding={'6px 7px'}
+                  fontSize={'10px'}
+                  borderRadius={'4px'}
+                  border={'solid 1px #3e4252'}
+                  display={'inline-block'}
+                >
+                  Current Utilization
+                </Box>
+                <Box mb={'20px'} borderLeft={'1px dashed #484c58'}>
+                  <ReactApexChart
+                    options={{

```

Comment:
Don't we use apex charts already do we have existing models to reuse here orare they hardcoded in a similar fashion?